configure.in

```diff
@@ -241,25 +242,18 @@ case "${host_os}" in
                  mac_version_min="-mmacosx-version-min=10.6"
                  mac_sysroot="-isysroot $(xcrun --sdk macosx --show-sdk-path)"
               fi
+
+              dnl Pick which architectures to build for based on whether
+              dnl the arch is defined in cdefs.h
+              mac_arches=""
+              for arch in x86_64 arm64
```

Comment:
If we are still supporting old OSes perhaps don't drop support for x86?

Reply:
Ross and I talked.
The previous code did not support x86. And 10.6 support x86_64.